pkg/ddc/juicefs/transform_fuse.go

```diff
@@ -411,3 +424,75 @@ func (j *JuiceFSEngine) genFormatCmd(value *JuiceFS, config *[]string) {
 	cmd := append([]string{common.JuiceCliPath, "auth"}, args...)
 	value.Configs.FormatCmd = strings.Join(cmd, " ")
 }
+
+// getQuota: get quota from string
+func (j *JuiceFSEngine) getQuota(v string) (int64, error) {
+	q, err := resource.ParseQuantity(v)
+	if err != nil {
+		return 0, fmt.Errorf("invalid quota %s: %v", v, err)
+	}
+	qs := q.Value() / 1024 / 1024 / 1024
```

Comment:
what if the quota is `3.5Gi`, can we support decimal quota values?

Reply:
no, quota is int64 and unit is GiB